prov/efa/docs/efa_rdm_protocol_v4.md

```diff
@@ -202,6 +204,19 @@ In the table, `type` is the packet type ID.
 `flags` is a set of flags each packet type uses to customize its behavior. Typically, it is used
 to indicate the existence of optional header(s) in the packet header. Each packet type defines its own flags.
 
+Protocol v4 define the following universal flag, which every packet type should use:
+
+Table: 1.4 a list of universal flags
+
+| Bit ID | Value | Name | Description | Used by |
+|-|-|-|-|
+| 15     | 0x8000 | CONNID_HDR | This packet has "connid" in header | extra request "connid header" (section 4.4) |
```

Comment:
Kind of hinted at this in a previous comment, but why does this start at 0x8000?

Reply:
Because each packet defines its own set of flags, which starts from the beginning. So I think the universal flag should start from the last bit, and going back wards.